test/test_regexp.py

```diff
@@ -0,0 +1,172 @@
+# ferny - asyncio SSH client library, using ssh(1)
```

Comment:
This commit should be squashed with the previous one IMHO.

Reply:
I separated it because it also tests existing functionality.  I'll split out the "add tests for error categorization" into a separate commit.

Reply:
I did the split, as described.